src/echo/web/logger.gleam

```diff
@@ -21,8 +22,11 @@ fn format_log_line(request: Request(a), response: Response(b)) -> String {
 
 pub fn middleware(service: Service(a, b)) -> Service(a, b) {
   fn(request) {
-    let response = service(request)
-    io.println(format_log_line(request, response))
-    response
+    service(request)
+      |> function.tap(fn(response) {
+        response
+          |> format_log_line(request, _)
+          |> io.println
+      })
```

Comment:
Let's revert this one, it's less clear what is happening as you have to learn about tap, and there's no existing pipeline so there's no clear advantage

Reply:
Thanks for the review 😄. Sure, I will revise this and fix formatting as well (as the CI caught it).